MekHQ/src/mekhq/campaign/parts/ProtomekLegActuator.java

```diff
@@ -72,7 +71,7 @@ public Money getStickerPrice() {
     @Override
     public boolean isSamePartType (Part part) {
         return part instanceof ProtomekLegActuator
-                && getUnitTonnage() == ((ProtomekLegActuator)part).getUnitTonnage();
+                && getUnitTonnage() == part.getUnitTonnage();
```

Comment:
Doublecheck that ProtomekLegActuator doesn't have implementation of getUnitTonnage()

Reply:
Same as above